logging/unit_tests/test_client.py

```diff
@@ -584,6 +637,9 @@ def create_scoped(self, scope):
         self._scopes = scope
         return self
 
+    def authorize(self, http):
+        pass
```

Comment:
Might want to add `_http` to the `_Credentials` class and then set it here so you can check what `http` is in tests if needed. Also would get rid of a lint error I think.